packages/components/src/components/NewModal/NewModal.stories.tsx

```diff
@@ -0,0 +1,147 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { NewModal as ModalComponent, NewModalProps, variables } from '../../index';
+import { ThemeProvider } from 'styled-components';
+import { intermediaryTheme } from '../../index';
+
+const Buttons = () => (
+    <>
+        <ModalComponent.Button>Primary Action</ModalComponent.Button>
+        <ModalComponent.Button variant="tertiary">Secondary Action</ModalComponent.Button>
+    </>
+);
+
+const meta: Meta = {
+    title: 'NewModal',
+    component: ModalComponent,
+    decorators: [
+        Story => (
+            <ThemeProvider theme={intermediaryTheme.light}>
```

Comment:
Are we going to be able to switch the theme somehow?

Reply:
Tried to do it but couldn't figure how. IMO we should skip the whole dual story preview thing and just make a theme switch in the Storybook.

Reply:
It's fine for now. In the future we could use 

![image](https://github.com/user-attachments/assets/c510b30a-07f1-4aec-9978-ebe16f75a4c8)

https://storybook.js.org/docs/essentials/toolbars-and-globals